src/proxnut/proxnut.py

```diff
@@ -72,7 +72,11 @@ def validate(self):
         # Validate Proxmox configuration
         logger.info("Validating Proxmox configuration...")
         if not self.target_machines:
-            raise ValidateError("No target machines configured")
+            # If no target machines configured, default to all nodes in cluster
+            logger.info("No target machines configured. Defaulting to all nodes in cluster.")
+            self.target_machines = self.proxmox_client.get_nodes()
```

Comment:
With this implementation, the nodes to be shut down are determined at startup, and there is a concern that newly added nodes may not be shut down if the system has been running for a long time. This behavior is difficult for users to understand, so it is better to acquire nodes at shutdown and shut down all acquired nodes.